Update steps for running WinAppDriver autonomously (#1368)
microsoft/WinAppDriver: Docs/RunningOnRemoteMachine.md

Therefore we need to follow the below steps on the remote machine for the Windows GUI tests to run in an autonomous fashion :
- **5.1**.	Setup Batch file to kill any old instances of WinAppDriver - This will be used in the KILLWAD scheduled task later 
-	*Name* : kill_winappdriver.bat
-	*Contents* : `taskkill /im WinAppDriver.exe /f`
- **5.2**.	Setup Batch file to start WinAppdriver  - This will be used in the StartWAD scheduled task later 
-	*Name* : LaunchWAD.bat
-	*Contents* : `cmd start /K "C:/Program Files (x86)/Windows Application Driver/WinAppDriver.exe" 10.x.xx.xx 4723/wd/hub`
-	*Note*: The IP address above (10.x.xx.xx) should be replaced with the local IP address of the server/remote machine
- **5.3**.	Setup Batch file to logout (without disconnecting) from the remote machine : 
-	*Name* : logout-rdp.bat

> -	*Contents*: `for /f "skip=1 tokens=3" %%s in ('query user %USERNAME%') do (%windir%\System32\tscon.exe %%s /dest:console C:\Install\QRes.exe /x 1920 /y 1080)`

Reviewer: what QRes.exe?

Reviewer 2: @licanhua - We can setup Qres (http://qres.sourceforge.net/) or another remote vnc tool to change the resolution of the target machine to match the local machine. My solution is loosely derived from here : https://stackoverflow.com/questions/35862217/screen-resolution-changes-for-the-remote-machine-after-safely-closing-the-rdp-se . 

Alternatively the user can skip setting up the destination resolution with QRes .

Author: @licanhua - We can setup Qres (http://qres.sourceforge.net/) or another remote vnc tool to change the resolution of the target machine to match the local machine. My solution is loosely derived from here : https://stackoverflow.com/questions/35862217/screen-resolution-changes-for-the-remote-machine-after-safely-closing-the-rdp-se .

Alternatively the user can skip setting up the destination resolution with QRes .

> -	*Contents*: `for /f "skip=1 tokens=3" %%s in ('query user %USERNAME%') do (%windir%\System32\tscon.exe %%s /dest:console C:\Install\QRes.exe /x 1920 /y 1080)`

Reviewer: ```suggestion
-	*Contents*: `for /f "skip=1 tokens=3" %%s in ('query user %USERNAME%') do (%windir%\System32\tscon.exe %%s /dest:console C:\Install\QRes.exe /x 1920 /y 1080)`
-	*Note*:  Qres (http://qres.sourceforge.net/) is a remote vnc tool to change the resolution of the target machine to match the local machine
```

CalculatorSession = (WindowsDriver)(new WindowsDriver(new URL("http://10.X.X.52:4723/wd/hub"), capabilities));
CalculatorSession.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
CalculatorResult = CalculatorSession.findElementByAccessibilityId("CalculatorResults");
 ```

>  5.WinAppDriver requires the Winappdriver server to be listening on the local host or IP address in order to perform the requisite UI interactions. While Appium generally takes care of this requirement on the local , for running on a server/remote machine, we need to perform a few extra steps as Winappdriver requires a GUI Output and the Winappdriver server running in an interactive shell. 

Reviewer 3: ```suggestion
 5. WinAppDriver requires the WinAppDriver server to be listening in order to perform the requisite UI interactions. While Appium generally takes care of this requirement on the local machine, running on a server/remote machine requires a few extra steps as WinAppDriver requires an interactive desktop session. 
```
